lib/ancestry/materialized_path.rb

```diff
@@ -75,7 +75,9 @@ def siblings_of(object)
     end
 
     def ordered_by_ancestry(order = nil)
-      if %w(mysql mysql2 sqlite sqlite3 postgresql).include?(connection.adapter_name.downcase) && ActiveRecord::VERSION::MAJOR >= 5
+      if %w(mysql mysql2).include?(connection.adapter_name.downcase) && ActiveRecord::VERSION::MAJOR >= 5
```

Comment:
I think that you can include sqlite in this treatment of the column. You probably also don't need the ActiveRecord version check as we're not doing anything fancy with ordering here.

Reply:
I agree.

I think that version check is to avoid using an arel named function in the `reorder()` method call

Since the statement is quite simple, we can avoid that.